independent-projects/tools/devtools-testing/src/main/java/io/quarkus/devtools/testing/WrapperRunner.java

```diff
@@ -104,7 +106,7 @@ public static int run(Path projectDir, Wrapper wrapper) {
 
     private static String getMavenSettingsArg() {
         final String mavenSettings = System.getProperty("maven.settings");
-        if (mavenSettings != null) {
+        if (mavenSettings != null && Files.exists(Paths.get(mavenSettings))) {
             return mavenSettings;
         }
```

Comment:
According to @aloubyansky it should be like:

```suggestion
        if (mavenSettings != null) {
            return Files.exists(Paths.get(mavenSettings)) ? mavenSettings : null;
        }
```

Reply:
@aloubyansky shouldn't we use `-s` if the file does not exist?